src/firmware/app/world/charger.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+/**
+ * This struct represents a robot capacitor charger from the perspective of firmware
+ */
+typedef struct Charger Charger_t;
+
+/**
+ * Create a charger with the given functions for interacting with it
+ *
+ * @param charge Charge the capacitor of the robot
+ * @param float Set capacitor to float
```

Comment:
Missing javadoc [in/out]

Reply:
https://github.com/UBC-Thunderbots/Software/issues/1650

Reply:
Thanks for creating the ticket.